app/models/manageiq/providers/redhat/infra_manager/api_integration.rb

```diff
@@ -10,6 +10,8 @@ module ManageIQ::Providers::Redhat::InfraManager::ApiIntegration
     process_api_features_support
   end
 
+  DEFAULT_PATH = '/ovirt-engine/api'.freeze
```

Comment:
is this used somewhere else than below? If not, I would just inline it (put the string where it's used)